### PR TITLE
Add GitHub Tips and Tricks

### PR DESCRIPTION
**Pull request by DunderRoffe**
Add text about tips and tricks specific to GitHub

Signed-off-by: Viktor Sjölind <viktor.sjolind@pelagicore.com>

### CODE REVIEW

File: docs/chapters/workflow/github-tips-and-tricks.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+on other platforms as well but no guarantees are made.
+
+Managing Pull requests
+----------------------
+Using the Pull requests view located in the top bar on the GitHub page
+(https://github.com/pulls) it is possible to view all Pull requests where:
+
+* Your user is the creator.
+* Your user is Assigned.
+* Your user is mentioned
```
Comment (GordanM) [inline docs/chapters/workflow/github-tips-and-tricks.rst]: Missing `.` for consistency.

File: docs/chapters/workflow/github-tips-and-tricks.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Following are tips and tricks specific to GitHub. This might work similarly
+on other platforms as well but no guarantees are made.
+
+Managing Pull requests
+----------------------
+Using the Pull requests view located in the top bar on the GitHub page
+(https://github.com/pulls) it is possible to view all Pull requests where:
+
+* Your user is the creator.
+* Your user is Assigned.
```
Comment (GordanM) [inline docs/chapters/workflow/github-tips-and-tricks.rst]: Either capitalize the keyword or leave it in small caps for consistency.

**Review by GordanM — COMMENTED** [review]
Nice! Just two small nitpicks.

**Comment by DunderRoffe** [comment]
Done

**Review by GordanM — APPROVED** [review]
Bueno.

**Review by breakreturn — APPROVED** [review]

**Pull request merged**